testdata/.tranor/envs.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
This is indeed just for test. This change doesn't cover the setup process yet. See my comment below for some ideas on how to install it.